pkg/flowaggregator/exporter/ipfix.go

```diff
@@ -144,7 +150,8 @@ func (e *IPFIXExporter) UpdateOptions(opt *options.Options) {
 		e.observationDomainID = genObservationDomainID(e.clusterUUID)
 	}
 	e.templateRefreshTimeout = opt.TemplateRefreshTimeout
-	klog.InfoS("New IPFIXExporter configuration", "collectorAddress", e.externalFlowCollectorAddr, "collectorProtocol", e.externalFlowCollectorProto, "sendJSON", e.sendJSONRecord, "domainID", e.observationDomainID, "templateRefreshTimeout", e.templateRefreshTimeout)
+	e.maxIPFIXMsgSize = int(opt.Config.FlowCollector.MaxIPFIXMsgSize)
+	klog.InfoS("New IPFIXExporter configuration", "collectorAddress", e.externalFlowCollectorAddr, "collectorProtocol", e.externalFlowCollectorProto, "sendJSON", e.sendJSONRecord, "domainID", e.observationDomainID, "templateRefreshTimeout", e.templateRefreshTimeout, "maxIPFIXMsgSize", e.maxIPFIXMsgSize)
 
 	if e.exportingProcess != nil {
 		e.exportingProcess.CloseConnToCollector()
```

Comment:
should it flush `bufferedExporter` first like `Stop()`?

Reply:
I would say it's reasonable not to flush in case of error?

Reply:
This is when user changes the configuration, right? If not flushed, could some records received in the last second be lost. My understanding this is similar to `Stop` + `Start`, so I wonder why we flush upon `Stop` but not here.

Reply:
you're right actually, I thought you left that comment in a different place. I'll make the change